### PR TITLE
Symfony 7 / PHP 8.3 compatibility

### PR DESCRIPTION
**Pull request by Vincz**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Tests pass?   | yes/no
| Documented?   | yes/no
| Fixed tickets | #... 
| License       | MIT

### CODE REVIEW

File: tests/Functional/App/config/conflictingValidatorNamespaces/config.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ imports:
     - { resource: ../config.yml }
 
 framework:
-    annotations: true
+    # annotations: true
```
Comment (mcg-web) [inline tests/Functional/App/config/conflictingValidatorNamespaces/config.yml:5]: please remove useless

File: tests/Functional/App/config/conflictingValidatorNamespaces/config.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@ imports:
...
     validation:
         enabled: true
-        enable_annotations: true
+        # enable_annotations: true
```
Comment (mcg-web) [inline tests/Functional/App/config/conflictingValidatorNamespaces/config.yml:8]: please remove useless

File: tests/Functional/App/config/typeShorthand/config.yml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 imports:
     - { resource: ../config.yml }
 framework:
-    annotations: true
+    # annotations: true
```
Comment (mcg-web) [inline tests/Functional/App/config/typeShorthand/config.yml:4]: please remove useless

File: tests/Functional/App/config/typeShorthand/config.yml [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
     validation:
         enabled: true
-        enable_annotations: true
+        # enable_annotations: true
```
Comment (mcg-web) [inline tests/Functional/App/config/typeShorthand/config.yml:7]: please remove useless

File: tests/Functional/App/config/validator/config.yml [hunk 5]
```diff
@@ -2,10 +2,10 @@ imports:
     - { resource: ../config.yml }
 
 framework:
-    annotations: true
+    #annotations: true
```
Comment (mcg-web) [inline tests/Functional/App/config/validator/config.yml:5]: please remove useless

File: tests/Functional/App/config/validator/config.yml [hunk 6]
```diff
@@ -2,10 +2,10 @@ imports:
...
     validation:
         enabled: true
-        enable_annotations: true
+        #enable_annotations: true
```
Comment (mcg-web) [inline tests/Functional/App/config/validator/config.yml:8]: please remove useless

**Review by mcg-web — DISMISSED** [review]
Thank you @Vincz ! Some cleanup and this can be merge !

**Comment by Vincz** [comment]
@mcg-web Thank boss! Fixed & merged :) Shall we publish a new version?

**Comment by jerowork** [comment]
@mcg-web Could you create a new release, including these changes? :) Thanks!

**Comment by luckyraul** [comment]
@Vincz graphiql also please

**Pull request merged**